openai_sample.py

```
@@ -0,0 +1,114 @@
+from openai import OpenAI, AsyncOpenAI
+import os
+import sys
+
+from monocle_apptrace import start_trace, stop_trace
+from monocle_apptrace.instrumentation.common.wrapper import atask_wrapper, task_wrapper
+os.environ["OPENAI_API_KEY"] = ""
+sys.path.append(os.path.join(os.path.dirname(__file__), "src"))
+from monocle_apptrace import setup_monocle_telemetry
+
+setup_monocle_telemetry(
+    workflow_name="openai_sample",
+    wrapper_methods=[
+    ],
+)
+
+async def async_main(): 
+    # Check if the OpenAI API key is
+    # set in the environment variables
+    if "OPENAI_API_KEY" not in os.environ:
+        print("Please set the OPENAI_API_KEY environment variable.")
+        return
```

should logger be not used ?? and throw exception instead of returning

---

this is a sample script, I am going to remove it soon

---

The file is removed